codegen/core/src/main/java/software/amazon/smithy/python/codegen/generators/StructureGenerator.java

```diff
@@ -309,7 +309,7 @@ private boolean hasDocs() {
     private void writeMemberDocs(MemberShape member) {
         member.getMemberTrait(model, DocumentationTrait.class).ifPresent(trait -> {
             String memberName = symbolProvider.toMemberName(member);
-            String docs = writer.formatDocs(String.format(":param %s: %s", memberName, trait.getValue()));
+            String docs = writer.formatDocs(String.format(":param %s: %s", memberName, trait.getValue())).replace("\n", "\n\t");
```

Comment:
Does this add a literal tab or 4 spaces? We should try to get everything in the generator to spaces, I'm not sure if the code formatter is handling that for us.